docs/data/sdk-middleware.md

```diff
@@ -102,6 +102,26 @@ Use an Middleware to modify event properties, transformation, filtering, routing
         });
         ```
 
+    ???code-example "Send event level groups using Ampli v1 (click to expand)"
```

Comment:
It is not rendered correctly for some reason. I tried to fix it by indenting this block but it's not working. If we cannot make it expandable, maybe we could at least fix the code blocks.
![image](https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/31029607/d7a14645-7388-49a7-bc8d-32ebac2c3352)